ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -530,6 +533,53 @@ namespace OrdinalAxis {
             return groupIntervalFactor;
         }
 
+        /**
+         * Get index of point inside the extended ordinal positions array.
+         *
+         * @private
+         * @param {number} val
+         *        The pixel value of a point.
+         *
+         * @param {Array<number>} [ordinallArray]
+         *        An array of all points available on the axis
+         *        for the given data set.
+         *        Either ordinalPositions if the value is inside the plotArea
+         *        or extendedOrdinalPositions if not.
+         *
+         * @return {number}
+         */
+        public getIndexOfPoint(
+            val: number,
+            ordinallArray: Array<number>
+        ): number {
+            const ordinal = this,
+                axis = ordinal.axis;
+            let firstPointVal = (axis.series[0].points && axis.series[0].points[0].x) || 0,
```

Comment:
If you are later looping through all of the series (including the first one) You don't need to assign this value to `(axis.series[0].points && axis.series[0].points[0].x)`. You can assign it to 0 right away. This value will later get overwritten.

By the way, can't we just use `axis.dataMin`?
@pawelfus do you have any better idea, how to get those values?